db/api/src/peers.rs

```diff
@@ -150,6 +150,12 @@ impl PeerStatus {
     pub fn get_quality(&self) -> f64 {
         self.quality
     }
+
+    /// Determines if this peer is still ignored.
+    #[inline]
+    pub fn is_ignored(&self, now: SystemTime, max_ignore: Duration) -> bool {
```

Comment:
Just from the type of passed in variables I would not know what this function should be doing. It's not immediately obvious how the system time and duration actually correlate and why system time is not obtained inside this function.